tests/webapps/test-war-hello/src/test/java/com/google/cloud/runtimes/jetty/tests/DeploymentITCase.java

```diff
@@ -0,0 +1,25 @@
+package com.google.cloud.runtimes.jetty.tests;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import com.google.cloud.runtime.jetty.testing.AppDeployment;
+import com.google.cloud.runtime.jetty.testing.HttpURLUtil;
+
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+
+public class DeploymentITCase
+{
+    @Test
+    public void testGet() throws IOException
+    {
+        HttpURLConnection http = HttpURLUtil.openTo(AppDeployment.SERVER_URI.resolve("/hello/"));
```

Comment:
We may need to implement URL fetch with retries because sometimes it takes a few seconds for the app to really be up after it's deployed.